__tests__/__unit__/Profiles.unit.test.ts

```diff
@@ -390,40 +389,40 @@ describe("Profile class unit tests", () => {
         expect(Profiles.getInstance().defaultProfile).toEqual(profileOne);
     });
 
-    it("should call listProfile", async () => {
-
-        Object.defineProperty(Profiles, "getInstance", {
-            value: jest.fn(() => {
-                return {
-                    allProfiles: [{name: "firstName"}, {name: "secondName"}],
-                    defaultProfile: {name: "firstName"},
-                    createNewConnection: jest.fn(()=>{
-                        return {};
-                    }),
-                    listProfile: jest.fn(()=>{
-                        return {};
-                    }),
-                };
-            })
-        });
-        const res = (await Profiles.createInstance(log)).listProfile();
-        expect(res).toEqual([profileOne, profileTwo]);
-    });
-
-    it("should call listProfile with invalid allProfiles", async () => {
-
-        Object.defineProperty(Profiles.getInstance, "listProfile", {
-            value: jest.fn(() => {
-                return {
-                    allProfiles: undefined,
-                    defaultProfile: undefined,
-                    map: jest.fn()
-                };
-            })
-        });
-        (await Profiles.createInstance(log)).listProfile();
-        expect(showErrorMessage.mock.calls.length).toBe(1);
-    });
+    // it("should call listProfile", async () => {
```

Comment:
We will need to figure out testing for this implementation.
And clean-up some other test files maybe?

Reply:
Are you thinking testing such as integration testing?. I will check the testing for more stuff to remove

Reply:
Removed all of the redundant tests. I didn't create any new ones as I still seem to have the coverage I need and the tests were for now removed code